FFXIVClientStructs/FFXIV/Client/Game/UI/PlayerState.cs

```diff
@@ -162,6 +162,12 @@ public unsafe partial struct PlayerState {
     /// <remarks> Use <see cref="IsOrchestrionRollUnlocked(uint)"/> </remarks>
     [FieldOffset(0x5F0), FixedSizeArray] internal FixedSizeArray99<byte> _unlockedOrchestrionRollBitmask;
 
+    [MemberFunction("83 FA ?? 72 ?? 33 C0 C3 8B C2 8B 84 81 ?? ?? ?? ?? C3 CC CC CC CC CC CC CC CC CC CC CC CC CC CC 83 FA")]
+    public partial int GetAttributeByIndex(PlayerAttribute attribute);
+
+    public int GetAttributeByIndex(uint attribute) =>
+        GetAttributeByIndex((PlayerAttribute)attribute);
+
```

Comment:
```suggestion
    public int GetAttribute(PlayerAttribute attribute) => Attributes[(uint)attribute];
```

No `CC` sig aka needing to span over multiple functions

Reply:
If people need to index integers that don't exist in the enum they can use `(PlayerAttribute)28` when calling the function